src/overlay005/fieldmap.c

```diff
@@ -400,24 +400,24 @@ static BOOL ov5_021D119C(FieldSystem *fieldSystem)
 static BOOL FieldMap_ChangeZone(FieldSystem *fieldSystem)
 {
     u32 v0;
-    u32 v1;
-    int v2, v3;
-    FieldOverworldState *v4;
+    u32 mapId;
```

Comment:
**polish:** We're working on cleaning up C89-style declarations to be more concise and declared when needed, as long as it matches. For example:

```diff
-    u32 mapId;
...
-    mapId = fieldSystem->location->mapId;
+    u32 mapId = fieldSystem->location->mapId;
```
I think these should match.